docs/streaming-kinesis-integration.md

```diff
@@ -32,7 +32,7 @@ A Kinesis stream can be set up at one of the valid Kinesis endpoints with 1 or m
 		import com.amazonaws.services.kinesis.clientlibrary.lib.worker.InitialPositionInStream
 
 		val kinesisStream = KinesisUtils.createStream(
-        	streamingContext, [Kinesis stream name], [endpoint URL], [checkpoint interval], [initial position])
+        	streamingContext, [Kinesis stream name], [endpoint URL], [checkpoint interval], [initial position], [credentials])
```

Comment:
might be good to include some more details on what this is, exactly.

this is a bit confusing if you're thinking of this in terms of ACCESS_KEY_ID and SECRET_KEY (2 separate strings)

also, be sure to include any imports that are relevant to this code sample - similar to the InitialPositionInStream reference right before it.